demos/vc_issuer/provision

```diff
@@ -58,6 +60,16 @@ do
             shift; # shift past --issuer-canister & value
             shift;
             ;;
+        --issuer-derivation-origin)
+            DERIVATION_ORIGIN="${2:?missing value for '--issuer-canister'}"
+            shift; # shift past --issuer-canister & value
```

Comment:
```suggestion
            shift; # shift past --issuer-derivation-origin & value
```

Reply:
+ "missing value for ..." both here and in the next param

Reply:
Done.